setup.py

```diff
@@ -30,10 +30,11 @@
     readfiles_require +
     security_require + [
         'gocept.testing',
-        'mock>=0.8.0, < 4',
+        'mock>=0.8.0, < 4',  # PY2
         'plone.testing',
         'zope.testing',
-        'zipp < 2',
+        'zipp < 2',  # PY2
+        'rpdb',
     ])
```

Comment:
You have used `six` multiple times in this PR, so we should include it here as install requirement.

Reply:
Instead `rpdb` should be removed.